### PR TITLE
Updated RDS Doc for MSSQL dashboards

### PR DESCRIPTION
**Pull request by akhil-sumologic**
## Purpose of this pull request

Added new dashboards for MSSQL in Amazon RDS section.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/SUMO-239934

### CODE REVIEW

File: docs/integrations/amazon-aws/rds.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+The Sumo Logic Amazon RDS app dashboards provide visibility into the performance and operations of your Amazon Relational Database Service (RDS). Preconfigured dashboards allow you to monitor critical metrics of your RDS instance(s) or cluster(s) including CPU, memory, storage, network transmits and receive throughput, read and write operations, database connection count, disk queue depth, and more. CloudTrail Audit dashboards help you monitor activities performed on your RDS infrastructure. MySQL Logs dashboards helps you monitor database errors, slow queries, audit sql queries and generic activities. PostgreSQL logs dashboard help you to monitor database errors, slow queries, database security, and query execution timings. MSSQL Logs dashboards helps you monitor error logs, basic infrastructure details.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:13]:
```suggestion
The Sumo Logic Amazon RDS app dashboards provide visibility into the performance and operations of your Amazon Relational Database Service (RDS). Preconfigured dashboards allow you to monitor critical metrics of your RDS instance(s) or cluster(s) including CPU, memory, storage, network transmits and receive throughput, read and write operations, database connection count, disk queue depth, and more. CloudTrail Audit dashboards help you monitor activities performed on your RDS infrastructure. MySQL Logs dashboards helps you monitor database errors, slow queries, audit sql queries and generic activities. PostgreSQL logs dashboard help you to monitor database errors, slow queries, database security, and query execution timings. MSSQL Logs dashboards helps you monitor error logs and basic infrastructure details. 
```

File: docs/integrations/amazon-aws/rds.md [hunk 2]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
 * Monitor time comparison for number of queries executed and query execution time.
 
 <img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-PostgreSQL-Logs-Query-Execution-Time.png')} alt="Amazon RDS dashboard" />
+
+### 18. Amazon RDS - MSSQL Logs - Error Logs - Logon Analysis
+
+The **Amazon RDS - MSSQL Logs - Error Logs - Logon Analysis** dashboard provides details for error logs, including failed authentications, logon error. This dashboard relies on MSSQL error logs, which needs to be [enabled](#collect-amazon-rds-cloudwatch-logs) for Amazon MSSQL. To view the data on the panels you need to first ingest MSSQL logs into Sumo Logic.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:687]:
```suggestion
The **Amazon RDS - MSSQL Logs - Error Logs - Logon Analysis** dashboard provides information about the error logs, including failed authentications and logon errors. This dashboard relies on MSSQL error logs, which need to be [enabled](#collect-amazon-rds-cloudwatch-logs) for the Amazon MSSQL. You need to first ingest MSSQL logs into Sumo Logic to view data on the dashboard panels.
```

File: docs/integrations/amazon-aws/rds.md [hunk 3]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+
+Use this dashboard to:
+* Identify the authentication failures along with reason for user, client host, and client location that are used to connect.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:690]:
```suggestion
* Identify authentication failures, including the user, client host, and location used for connection.
```

File: docs/integrations/amazon-aws/rds.md [hunk 4]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+* Detect logon errors along with their associated error codes, severity levels, and states.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:691]:
```suggestion
* Detect logon errors, including error codes, severity levels, and states.
```

File: docs/integrations/amazon-aws/rds.md [hunk 5]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-MSSQL-Logs-Error-Logs-Logon Analysis.png')} alt="Amazon RDS dashboard" />
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:693]:
```suggestion
<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-MSSQL-Logs-Error-Logs-Logon Analysis.png')} style={{border:'1px solid gray’}} alt="Amazon RDS dashboard" />
```

File: docs/integrations/amazon-aws/rds.md [hunk 6]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+
+The **Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview** dashboard provides details for hardware, authentications mode, collation, process, recent termination of SQL server and recent creation of databases. This dashboard relies on MSSQL error logs, which needs to be [enabled](#collect-amazon-rds-cloudwatch-logs) and ingested into Sumo Logic.
+
+Use this dashboard to:
+* Get a high-level overview of your MSSQL infrastructure like instance type, version.
+* Get configuration details such as authentication mode, collation settings and process details.
+* Monitors `DBCC CHECKDB` checks
+* Track recent terminations of SQL Server instances and monitor the creation of new databases.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-MSSQL-Logs-Error-Logs-Infrastructure-Overview.png')} alt="Amazon RDS dashboard" />
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:705]:
```suggestion
<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-MSSQL-Logs-Error-Logs-Infrastructure-Overview.png')} style={{border:'1px solid gray’}} alt="Amazon RDS dashboard" />
```

File: docs/integrations/amazon-aws/rds.md [hunk 7]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+
+Use this dashboard to:
+* Identify the authentication failures along with reason for user, client host, and client location that are used to connect.
+* Detect logon errors along with their associated error codes, severity levels, and states.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-MSSQL-Logs-Error-Logs-Logon Analysis.png')} alt="Amazon RDS dashboard" />
+
+### 19. Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview
+
+The **Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview** dashboard provides details for hardware, authentications mode, collation, process, recent termination of SQL server and recent creation of databases. This dashboard relies on MSSQL error logs, which needs to be [enabled](#collect-amazon-rds-cloudwatch-logs) and ingested into Sumo Logic.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:697]:
```suggestion
The **Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview** dashboard provides about the hardware, authentication mode, collation, process, recent termination of SQL server, and recent creation of databases. This dashboard relies on MSSQL error logs, which need to be [enabled](#collect-amazon-rds-cloudwatch-logs) and ingested into Sumo Logic.
```

File: docs/integrations/amazon-aws/rds.md [hunk 8]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+
+Use this dashboard to:
+* Get a high-level overview of your MSSQL infrastructure like instance type, version.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:700]:
```suggestion
* Get a high-level overview of your MSSQL infrastructure like instance type and version.
```

File: docs/integrations/amazon-aws/rds.md [hunk 9]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+* Get configuration details such as authentication mode, collation settings and process details.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:701]:
```suggestion
* Get configuration details such as authentication mode, collation settings, and process details.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fix

File: docs/integrations/amazon-aws/rds.md [hunk 10]
```diff
@@ -676,3 +681,25 @@ Use this dashboard to:
...
+* Monitors `DBCC CHECKDB` checks
+* Track recent terminations of SQL Server instances and monitor the creation of new databases.
```
[the remark this answers is not in the document]
Comment (akhil-sumologic) [reply docs/integrations/amazon-aws/rds.md:703]: > Track recent terminations of SQL Server instances and monitor the creation of new databases.

what is correction here ?

[the remark this answers is not in the document]
Comment (JV0812) [reply docs/integrations/amazon-aws/rds.md:721]: Nothing to change here.

**Pull request merged**